indico/modules/events/registration/models/forms.py

```diff
@@ -32,10 +32,11 @@
 
 
 class ModificationMode(RichIntEnum):
-    __titles__ = [None, L_('Until modification deadline'), L_('Until payment'), L_('Never')]
+    __titles__ = [None, L_('Until modification deadline'), L_('Until payment'), L_('Until Approved'), L_('Never')]
     allowed_always = 1
     allowed_until_payment = 2
-    not_allowed = 3
+    allowed_until_approved = 3
```

Comment:
You can't re-number the enum like this - doing so would require updating the value for ALL existing regforms. BTW, regardless of this, you need an alembic revision to recreate the CHECK constraint on the column where this enum is used to allow the new value.